apps/web/components/Shell.tsx

```diff
@@ -57,9 +57,10 @@ function useRedirectToLoginIfUnauthenticated() {
   const { data: session, status } = useSession();
   const loading = status === "loading";
   const router = useRouter();
+  const shouldDisplayUnauthed = router.pathname.startsWith("/apps");
```

Comment:
instead of hard-coding`/apps` we can introduce a prop on `Shell` to determine whether it should render when unauth or not

Reply:
Proposed #2437 for your reference

Reply:
As for now we don't plan to add any more public apps. We can do that when that changes.